static/templates/actors/character/sidebar/class-dc.html

```diff
@@ -8,7 +8,7 @@ <h2>{{#if (gt classDCs.dcs.length 1)}}{{localize "PF2E.Actor.Character.ClassDC.P
     {{#each classDCs.dcs as |classDC|}}
         <div class="roll-data difficulty-checks class-dc{{#unless classDC.primary}} secondary{{/unless}}">
             <h3>
-                {{classDC.value}}
+                {{classDC.dc}}
```

Comment:
What about the modifiers tooltip?

Reply:
Still seems to work the same
![image](https://user-images.githubusercontent.com/1286721/192169125-85fb8689-7f7a-47b0-abe5-b682573a4245.png)